SimplioOfficial/simplio-app
# Task/116 create utxo wallet

# Overview

Initilizing sio_core library with generated mnemonic. Mnemonic is encrypted by default and decrepted after verifying pin. User can fetch available crypto assets from our API and filter them by `name` and `ticker`. Assets are grouped by asset type that is defined by `assetId` (Asset ID is internally defined numeric value representing and any type of asset). User can add a specific network wallet (An asset that lives on a specific blockchain network).

## Test cases:

1. Start the app and login. You should be prompt to set 4 digit pin.
2. After setting and submitting newly set pin, you should be redirected to a success page.
3. click on `go to app` button.
4. After setting pin there should not be any pin verification however if application is refreshed you will be asked to insert a pin
5. Go to inventory and click on search. There should be a loading animation and then assets should be listed.
6. Open and asset and its networks should be showed in dropdown.
7. Click on plus button. Note - there is no filtering at this moment.
8. Close the search page and you should se a newly created asset wallet.
9. Close the app and enter it again. Asset wallet should be persistent
10. Go to search and filter an asset by name or ticker. The list should be filtered.
